OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/pdf/PDFDisplay.java

```diff
@@ -36,9 +36,13 @@ public JPanel drawPreviewOnJPanel() {
         Image generatedImage = PDFPreviewBuilder.buildPDFPreview(selectedPage, viewZoom);
         
         if (generatedImage != null) {
-            ImageIcon icon = new ImageIcon(generatedImage);
-            JLabel picLabel = new JLabel(icon);
-            pdfImagePanel.add(picLabel);
+            try {
+                ImageIcon icon = new ImageIcon(generatedImage);
+                JLabel picLabel = new JLabel(icon);
+                pdfImagePanel.add(picLabel);
+            } catch (OutOfMemoryError oome) {
```

Comment:
Yeah, I'll go in an remove it


On Thu, Aug 18, 2016 at 3:45 AM, Sebastian Brudziński <
notifications@github.com> wrote:

> In OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/
> pdf/PDFDisplay.java
> <https://github.com/sebbrudzinski/Open-LaTeX-Studio/pull/137#discussion_r75261635>
> :
>
> > @@ -36,9 +36,13 @@ public JPanel drawPreviewOnJPanel() {
> >          Image generatedImage = PDFPreviewBuilder.buildPDFPreview(selectedPage, viewZoom);
> >
> >          if (generatedImage != null) {
> > -            ImageIcon icon = new ImageIcon(generatedImage);
> > -            JLabel picLabel = new JLabel(icon);
> > -            pdfImagePanel.add(picLabel);
> > +            try {
> > +                ImageIcon icon = new ImageIcon(generatedImage);
> > +                JLabel picLabel = new JLabel(icon);
> > +                pdfImagePanel.add(picLabel);
> > +            } catch (OutOfMemoryError oome) {
>
> So, can we remove this catch now?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sebbrudzinski/Open-LaTeX-Studio/pull/137/files/f823030ef76becab3cb924455ad2916bad99a496#r75261635>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAuhv4lWqz6TsGGYjJ6w1IhU0ogkzaUMks5qhA2YgaJpZM4JermW>
> .
>